source/blog/2017-10-13-ostree-reboot-saga.html.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Rolling Back Bad Kernel Updates
+author: jberkus
+date: 2017-10-13 14:00:00 UTC
+published: true
+comments: false
+tags: atomic, fedora, ostree
+---
+
+One of main features of Atomic Host is the ability to roll back bad updates.  This includes bad kernel updates.  Since Fedora Atomic follows Linux kernel releases fairly closely, we've seen a few of these, such as the [one which took out my minnowboard test cluster on August 22](/blog/2017/09/fedora-atomic-26-120/).
+
+READMORE
+
+This is my rolling test cluster, so it already had Fedora Atomic on it, originally the August 6th release:
+
+```
+-bash-4.4# rpm-ostree status
+State: idle
+Deployments:
+● fedora-atomic-26:fedora/26/x86_64/atomic-host
+                   Version: 26.101 (2017-08-06 21:27:14)
+                    Commit: f6331bcd14577e0ee43db3ba5a44e0f63f74a86e3955604c20542df0b7ad8ad6
+              GPGSignature: Valid signature by E641850B77DF435378D1D7E2812A6B4B64DAB85D
+
+  fedora-atomic-26:fedora/26/x86_64/updates/atomic-host
```

Comment:
The second deployment here can be confusing given that it's more recent than your currently booted one. Maybe just scrub it from the output?

Edit: yeah, basically the same content as after `cleanup -p` below. ⬇️

Reply:
Huh?  It's supposed to be more recent.  That's the point.

Reply:
The post demonstrates going from `26.101` to `26.110` and then eventually `26.114`, right? The fact that `26.114` is already there from the beginning breaks the illusion that these steps and outputs come from that timeline. Not a big deal of course, feel free to keep it here as well if you'd like!

Reply:
Oh!  No, that's a paste fail.  It wouldn't make any sense for them to be in that order regardless.  Fixed, thanks.